callautomation_appointmentbooking/exception/exception_handler.py

```diff
@@ -0,0 +1,23 @@
+from flask import json
+from flask.wrappers import Response
+from callautomation_appointmentbooking.exception.call_automation_exception import CallAutomationException
```

Comment:
```suggestion
from exception.call_automation_exception import CallAutomationException
```
and the same for the other files